userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -0,0 +1,172 @@
+= Managing Metrics with New Relic
+
+:imagesdir: https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws
+
+
+== Single Tier and Multitier Implementations
```

Comment:
Please change "single tier" to "single-tier" throughout. Same for "multitier" and "multi-tier." I've already made these changes for most of our documents. (The Datadog one got reverted when I went through an epic battle with git/GitHub recently, but I'll fix that.)

Reply:
done